client/components/LearnerViewDirective.js

```diff
@@ -284,6 +368,14 @@ tie.directive('learnerView', [function() {
           }
         };
 
+        $scope.navigateToQuestion = function(index) {
+          if ($scope.unlockedQuestions[index]) {
+            $scope.currentQuestionIndex = index;
+            var questionId = $scope.questionIds[$scope.currentQuestionIndex];
+            loadQuestion(questionId, questionSet.getIntroductionParagraphs());
```

Comment:
This is interesting (the fact that we pass intro paragraphs regardless of whether the question is the first one or not). I guess it makes sense, though it doesn't necessarily match history. Happy to go with it for now.